certif/app/models/session.js

```diff
@@ -58,10 +58,6 @@ export default class Session extends Model {
     return `${ENV.APP.API_HOST}/api/sessions/${this.id}/certification-candidates/import`;
   }
 
-  get urlToDownloadSessionIssueReportSheet() {
-    return ENV.urlToDownloadSessionIssueReportSheet;
-  }
```

Comment:
**Suggestion** : comme il s'agit d'une url je propose plutôt de le déplacer dans le service url et de gérer selon la current-lang de l'utilisateur.


**url.js**

get sessionIssueReportSheetUrl() {

  const currentLanguage = this.intl.t('current-lang');
  return currentLanguage === 'fr' ? 'https://cloud.pix.fr/s/B76yA8ip9Radej9/download' : 'https://cloud.pix.fr/s/ro7jHtsZZbY5SCX/download';
 
 }

Reply:
Je n'avais pas connaissance de ce service ! Merci !

Reply:
done